src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1192,10 +1192,11 @@ private void OnTopologyChangeInternal(ClusterTopology topology, Dictionary<strin
         {
             if (_lastClusterTopologyIndex < topology.Etag)
                 _lastClusterTopologyIndex = topology.Etag;
+            
+            _ = ClusterRequestExecutor.UpdateTopologyAsync(new RequestExecutor.UpdateTopologyParameters(new ServerNode() {ClusterTag = NodeTag, Url = Server.WebUrl}));
```

Comment:
do we want to have Debug Tag here?

Reply:
Extra info could be useful, no?

Reply:
I think so, that is why I've added my comment :)

Reply:
Woops I confused it with the other place where I already put a debugTag. Added.